docs/EdgeApps.md

```diff
@@ -266,27 +261,52 @@ The `homepage_url` field is a URL directing to the homepage of the Edge App.
 #### Auth
 
 The `auth` field is optional and is used to configure Edge App authentication. It includes the following subfields:
+
 - **auth_type:** Specifies the type of authentication. Possible values are:
   - `basic`: Basic authentication.
   - `bearer`: Bearer token authentication.
 - **global:** A boolean value that determines whether the authentication is global (`true`) or local (`false`).
 
 For Basic authentication, the following settings are generated:
+
 - `screenly_basic_auth_username`
 - `screenly_basic_auth_password`
 
 For Bearer authentication, the following setting is generated:
+
 - `screenly_bearer_token`
 
 These settings must be configured using the `screenly edge-app setting set` command.
 
 #### Ready Signal
 
-The `ready_signal` field is an optional boolean. When set to `true`, the Edge App must send a signal to the Screenly Player indicating it is ready to be displayed.
+The `ready_signal` field is an optional boolean parameter in the Edge App configuration.
+
+##### Purpose
+
+When set to `true`, it enables a mechanism for the Edge App to control when it gets rendered on the Screenly Player.
+
+##### Use Case
+
+This feature is particularly useful for apps that require initial loading or data preparation before they're ready to be shown. For example, an app might need to fetch data or complete some initialization process.
```

Comment:
Or heavy rendering for things like dashboards. Reference how we use this in the PowerBI Edge App.